src/poll/poll.dtos.ts

```diff
@@ -16,6 +16,11 @@ import { Transform, Type } from 'class-transformer';
 import { ObjectId } from 'mongodb';
 import constants from '../common/constants';
 import { IsAfterNow } from '../common/isAfterNowConstraint'
+import { DiscordEmbedFieldLength } from './discordEmbedFieldLengthConstraint'
+
+// Maximum number of fields to be used by poll_options and role_restrictions combined
+// Discord limit is 25, leaving 5 fields for date, vote count, strategy name, and misc
+const MAX_EMBED_FIELD_LENGTH = 20;
```

Comment:
So as we add more fields in the future, we would need to modify this

Reply:
Yes